src/components/ContactUs/ContactUsForm/ContactUsForm.tsx

```diff
@@ -56,18 +41,38 @@ const ContactUsForm: React.FC = () => {
         services: selectedServices
           .map((service) => intl(service.name))
           .join(', '),
-      }).then(() => {
-        resetForm()
-        setSelectedServices([])
       })
+        .then((result) => {
+          if (result.success) {
+            resetForm()
+            setSelectedServices([])
+            toast.success(intl('contact.us.form.toast.success'), {
+              id: toastId,
+              duration: 2000,
+              icon: <FaCheckCircle size={20} />,
+              position: 'top-center',
+              className: 'toast-custom toast-success',
+            })
+          } else {
+            throw new Error('Error al enviar')
+          }
+        })
+        .catch(() => {
+          toast.error(intl('contact.us.form.toast.error'), {
+            id: toastId,
+            duration: 2000,
+            icon: <FaExclamationCircle size={20} />,
+            position: 'top-center',
+            className: 'toast-custom toast-error',
```

Comment:
This doesn't belong in the `else`? `Catch `captures an error that arises in the `then`.

Reply:
result.success returns a boolean value. If the return value is false, I think generating an error to catch it might be more optimal.
We have to discuss that